web/src/utils/schema.ts

```diff
@@ -0,0 +1,38 @@
+import type { JSONSchema } from "@apidevtools/json-schema-ref-parser";
+
+/**
+ * Applies various miscellaneous fixes/massages the Dandiset metadata JSON Schema
+ * to make it compatible with VJSF.
+ *
+ * @param schema The Dandiset metadata JSON Schema
+ */
+export function fixSchema(schema: JSONSchema) {
+  let deepCopySchema = JSON.parse(JSON.stringify(schema));
+
+  // TODO: Fix this upstream in the schema
+  deepCopySchema['properties']['identifier']['pattern'] = '^DANDI:\\d{6}$'
+  deepCopySchema['$defs']['Project']['properties']['wasAssociatedWith']['items']['oneOf'][2]['properties']['identifier']['pattern'] = '^RRID:.*'
+
+  // vjsf renders < and > as HTML tags, which breaks rendering of the schema
+  // in places like help text. For example, given this description for a field:
+  //
+  // {
+  //   ...
+  //   "description": "This relation should satisfy: dandiset <relation> resource.",
+  //   ...
+  // }
+  //
+  // vjsf will render it as raw HTML, causing the < and > to be interpreted like this:
+  //
+  // <p>This relation should satisfy: dandiset <relation>resource</relation>.</p>
+  //
+  // To fix this, we escape the < and > characters that look like HTML tags.
+  deepCopySchema = JSON.parse(
+    JSON.stringify(schema)
+      .replace(
+        /<\/?[a-zA-Z][^>\s]*[^>]*>/g, t => t.replace(/</g, '\\\\<').replace(/>/g, '\\\\>')
+      )
+  );
```

Comment:
Because you're assigning `deepCopySchema` here, but passing in `schema` to `JSON.stringify`, aren't the above two assignments erased? Should this be:
```suggestion
  deepCopySchema = JSON.parse(
    JSON.stringify(deepCopySchema)
      .replace(
        /<\/?[a-zA-Z][^>\s]*[^>]*>/g, t => t.replace(/</g, '\\\\<').replace(/>/g, '\\\\>')
      )
  );
```

Reply:
Good catch. Now I'm confused as to how the meditor was rendering properly without those fixes being applied. Maybe they aren't needed anymore. I'll file an issue to look into that more closely